app/parse.py

```diff
@@ -14,5 +19,27 @@ class Course:
     course_type: CourseType
 
 
+def create_single_course(soup: BeautifulSoup) -> Course:
+    name = soup.select_one(".typography_landingH3__vTjok").text.split()
+    course_type = (
+        CourseType.PART_TIME if len(name) == 3 else CourseType.FULL_TIME
+    )
+
+    return Course(
+        name=name[1],
+        short_description=soup.select_one(
+            ".CourseCard_flexContainer__dJk4p"
+        ).text,
+        course_type=course_type
+    )
+
+
+def get_single_data(soup: BeautifulSoup) -> list:
```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]